docs/hub/api.md

```diff
@@ -224,7 +224,8 @@ Payload:
 ```js
 payload = {
     "fromRepo" : "namespace/repo_name",
-    "toRepo" : "namespace2/repo_name2"
+    "toRepo" : "namespace2/repo_name2",
+    "type": "dataset",
```

Comment:
In the other examples, `"type":"type"` instead of a specific `type` like `dataset`. I wonder if it might be better to just leave it as `type` to signal that it can also be a `model` or `space`?

```suggestion
    "type": "type",
```

Reply:
Good catch. I've pushed a new commit to align how we are setting this. I prefer to have `"type": "model"` as an example and explain in words what are the possible values.